docs/quickstart/create-a-local-test-network.md

```diff
@@ -85,117 +49,3 @@ cd $GOPATH/src/github.com/ava-labs/avalanchego
 ```sh
```

Comment:
Isn't it a bit redundant to copy the same command 5 times here? Would be easier to copy and past once

Reply:
This may be valid, but @holisticode did not change this.